tests/microos/journal_check.pm

```diff
@@ -16,46 +16,53 @@ use warnings;
 use testapi;
 use microos;
 use version_utils 'is_opensuse';
+use Mojo::JSON qw(decode_json);
+
+sub parse_bug_refs {
+    my $res     = shift;
+    my $product = get_var('DISTRI');
+    my $version = get_var('VERSION');
```

Comment:
```suggestion
    my $product = get_required_var('DISTRI');
    my $version = get_required_var('VERSION');
```

Otherwise it breaks because of  `undef`